### PR TITLE
New version

### PR DESCRIPTION
**Pull request by Vsevolod-Livinskij**
Add aligned attribute and dead variables pruning

### CODE REVIEW

File: src/statistics.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
+Copyright (c) 2015-2017, Intel Corporation
```
Comment (dbabokin) [inline src/statistics.cpp:2]: 2017?

File: src/statistics.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) 2015-2017, Intel Corporation
```
Comment (dbabokin) [inline src/statistics.h:2]: Same here, 2017?

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -40,7 +40,7 @@ set(LIB_SRCS
     "type.cpp"
     "type.h"
     "utils.cpp"
-    "utils.h")
```
Comment (dbabokin) [inline src/CMakeLists.txt:43]: New files names need to be in quotes and in alphabetical order.

Comment (Vsevolod-Livinskij) [reply src/CMakeLists.txt]: Fixed

File: src/statistics.h (hunk 2 above)
Comment (Vsevolod-Livinskij) [reply src/statistics.h]: Fixed

File: src/statistics.cpp (hunk 1 above)
Comment (Vsevolod-Livinskij) [reply src/statistics.cpp]: Fixed

**Pull request merged**